src/blocks/card.css

```
@@ -0,0 +1,91 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  justify-content: center;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+  margin-bottom: 40px;
+}
+
+.card__product-review {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__reviews-count {
+  color: #060b35;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  align-self: center;
+}
+
+.card__product-name {
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: #616070;
+  margin-bottom: 16px;
+  align-self: flex-start;
+}
+
+.card__btn {
+  color: white;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  width: 100%;
+  height: 40px;
+  text-align: center;
+  line-height: 40px;
+  font-size: 14px;
+}
```

without flex-box container you need write display: block for button
Also remove width 100% - it will be default value
```suggestion
.card__btn {
	display: block
  height: 40px;
  
  background-color: #00acdc;
  border: 1px solid #00acdc;
  border-radius: 5px;

  text-align: center;
  line-height: 40px;
  font-size: 14px;
  color: white;
}
```